app/uk/gov/nationalarchives/omega/editorial/views/editSet.scala.html

```diff
@@ -53,13 +53,13 @@
       head = Some(
           Seq(
               HeadCell(
-                  content = Text("CCR")
+                  content = Text(messages("edit-set.record.edit.ccr"))
```

Comment:
(Refers to line 28)

This might be out of scope, but: Why are we sometimes passing in messages that we could resolve within the view?

Reply:
I agree that we should be resolving messages within the views, however I think this would require a lot of refactoring to do consistently and would be out of scope.

Reply:
Agreed - one day.